recipes/phytop/src/small_tools.py

```
@@ -0,0 +1,358 @@
+		if os.path.getsize(ckgfile) == 0:
+			return True
+		data = []
+		with open(ckgfile, 'rb') as f:
+			while True:
+				try: dat = pickle.load(f)
+				except EOFError: break
+				data += [dat]
+		return data
+	return False
+def tr_numeric(val):
+	try: return int(val)
```

Ruff

Multiple statements on one line (colon)

(E701)